config/environments/production.rb

```diff
@@ -63,4 +63,6 @@
     config.developer_emails = emails.split(",").map(&:downcase).map(&:strip)
   end
 
+  config.auth_url = 'http://staging.sportenter.co.il'
```

Comment:
we need to add dynamic urls. (we need to support sportendter.co.il and staging.co.il)